test/README.md

```diff
@@ -50,6 +50,8 @@ AWS_ACCESS_KEY_ID=XXX AWS_SECRET_ACCESS_KEY=xxx npm run integration-test-run-all
 
 _Note: Home folder is mocked for test run, therefore relying on `AWS_PROFILE` won't work. _ and _secret key_, need to be configured directly into env variables\_
 
+\_Note: Some integration tests depend on shared infrastructure stack (see below)\_
```

Comment:
If you'd like to apply formatting, I think you shouldn't escape the `_` here

Reply:
Good point :sweat_smile: , I've not understood the formatting on the previous line.

Reply:
No worries, I think the previous line has broken formating :grimacing:

Reply:
Formatting for the new line should be ok now :smile: